### DISCUSSION TITLE
I want to disable strelka.

### DISCUSSION THREAD

**Discussion by 89sooner** [Security-Onion-Solutions/securityonion#12516] [General]
Hello, thank you in advance. I would like to stop the Strelka engine in the forward node.

I am currently using Suricata for the purpose of IDS, and everything else has been disabled, but I don't know how to disable Strelka.

Please help me.

**Comment by cm-ops** [comment]
Change the below in SOC to `false`

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/63416267/1e61546b-3ca8-48c2-8e25-f2a440434947)

**Reply by 89sooner** [reply]
Thank you! 
but my security onion version is 2.3.210..

**Reply by cm-ops** [reply]
In 2.3 that setting is in `/opt/so/saltstack/local/pillar/global.sls`.

**Comment by TOoSmOotH** [comment]
> Thank you! but my security onion version is 2.3.210..

That would have been helpful information to include in your original post. 2.3.X support ends in 3 weeks so I would really look into upgrading.
